### PR TITLE
mageutils:feat - Adding new method to add/remove tag

### PR DESCRIPTION
**Pull request by wiliansilvazup**
In this commit I added the new method CreateAndPushTag for create and
up new tag. And other new method RemoveTag for delete local and in
origin tag.

Signed-off-by: Wilian Gabriel <wilian.silva@zup.com.br>

### CODE REVIEW

File: pkg/utils/mageutils/release.go [hunk 1]
```diff
@@ -44,6 +44,32 @@ func CreateLocalTag(tag string) (err error) {
 	return sh.RunV("git", "tag", "-s", tag, "-m", "release "+tag)
 }
 
+// CreateAndPushTag executes "git", "tag", "-s", tag, "-m", "release "+tag
+// and after executes "git", "push", "--tags"
```
Comment (matheusalcantarazup) [inline pkg/utils/mageutils/release.go:48]: Can we change this doc to something like this?
```
CreateAndPushTag create and push a new given tag.
```

I don't think that is clear document a function with the code it executes.

File: pkg/utils/mageutils/release.go [hunk 2]
```diff
@@ -44,6 +44,32 @@ func CreateLocalTag(tag string) (err error) {
...
+
+	if err := sh.RunV("git", "tag", "-s", tag, "-m", "release "+tag); err != nil {
+		return err
+	}
+
+	return sh.RunV("git", "push", "--tags")
+}
+
+// RemoveTag executes "git", "tag", "-d", tag
+// and after executes "git", "push", "--delete", "origin", tag
```
Comment (matheusalcantarazup) [inline pkg/utils/mageutils/release.go:61]: Same here:
```
RemoveTag remove the given tag locally and from origin remote.
```

**Review by matheusalcantarazup — CHANGES_REQUESTED** [review]

File: pkg/utils/mageutils/release.go (hunk 1 above)
Comment (wiliansilvazup) [reply pkg/utils/mageutils/release.go:48]: OK, but this methods will be different of the others, I can continue?

Comment (matheusalcantarazup) [reply pkg/utils/mageutils/release.go:48]: Oh, I didn't notice that all docs have this pattern. Maybe would be good leave it like this for consistency, or maybe we can merge the both?

```
CreateAndPushTag create and push a new given tag executing 
"git tag -s tag -m release+tag" and "git push --tags"
```

Comment (wiliansilvazup) [reply pkg/utils/mageutils/release.go:48]: Okay :)

**Review by nathanmartinszup — APPROVED** [review]

**Review by matheusalcantarazup — APPROVED** [review]

**Pull request merged**